charts/tempo-distributed/values.yaml

```diff
@@ -244,6 +244,9 @@ querier:
   extraVolumeMounts: []
   # -- Extra volumes for querier deployment
   extraVolumes: []
+  config:
```

Comment:
Let's mimic the structure of the actual config to future proof us a bit:
```
config:
  frontend_worker:
    grpc_client_config:
```

Reply:
@joe-elliott done as requested

Reply:
@joe-elliott done as requested